beamline/ui/main.py

```diff
@@ -16,7 +16,7 @@
 import wx
 
 
-def run(debug=True, icon=None):
+def run(icon=None):
```

Comment:
.. versionchanged:: 2.0.0
    removed kwarg `debug=True`

Reply:
Good point, I've just added the note.